src/installer/installer.go

```diff
@@ -95,6 +95,7 @@ func (i *installer) InstallNode() error {
 	i.Config.Device = i.ops.EvaluateDiskSymlink(i.Config.Device)
 	err := i.cleanupInstallDevice()
 	if err != nil {
+		i.UpdateHostInstallProgress(models.HostStageStartingInstallation, fmt.Sprintf("Failed to cleanup install device %s", i.Device))
```

Comment:
This is user facing. We should soften the error

Reply:
What about "Non fatal error: could not clean install device %s, installation will continue" ?

Reply:
Agree, @wolfeallison PTAL

Reply:
For error messages, we try to follow the pattern description, reason, resolution.  
"Could not clean install device
The installation will continue. If the installation fails, clean the disk and try again."

What do you think of this? @javipolo @eranco74

Reply:
I like it :+1: , let's see how @eranco74 feels about it